src/momento/responses/response.py

```diff
@@ -1,5 +1,44 @@
+from __future__ import annotations
+
 from abc import ABC
+from typing import Any, no_type_check
 
 
-class CacheResponse(ABC):
+class Response(ABC):
     """Parent of all responses"""
+
+    # We need to work with Any.
+    @no_type_check
+    def __repr__(self) -> str:
+        class_name = self.__class__.__qualname__
+        attributes = vars(self)
+
+        if not attributes:
+            return f"{class_name}()"
+
+        message = {k: self._truncate_value(v) for k, v in attributes.items()}
+
+        return f"{class_name}({message!r})"
```

Comment:
repr format is keyword like arguments. With this, a `CacheGet.Hit` renders as:

`CacheGet.Hit({'value_bytes':` b'value'})` but should be `CacheGet.Hit(value_bytes=b'value')`

Reply:
I can't get it to do that without a lot more work.

Reply:
Fixed in [67d3dea](https://github.com/momentohq/client-sdk-python/pull/258/commits/67d3dea8f1e876a5062801592254ed9fbe66c051)